docs/welcome/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+tags: [00. Welcome]
+---
+
+# Intro
+
+Studio is our next generation app for API design, modeling, and technical writing. 
+
+> A primary goal of Studio is to enrich, not replace, your existing workflows. It works offline, with folders and files on your computer, just like your favorite IDEs.
+
+Here is some of what it can do:
+
+- **OpenAPI v2 AND v3:** Form based design mode means you don't need to be an OpenAPI expert to get started. Studio also supports OpenAPI autocomplete in write mode, a read mode to visualize HTTP operations and models, mocking, and linting for OpenAPI v2 and v3.
+- **Standalone JSON Schema Modeling:** Studio encourages you to split your models into separate files, and then makes it easy to create `$refs` between them.
```

Comment:
This seems like its confusing two different things.

1. We encourage and support OpenAPI users splitting their models out into files with $refs
2. We support standalone JSON Schema completely outside of an OpenAPI context for people who do that

1 is not JSON Schema its "OpenAPI Schema Objects" and confusing those things is asking for trouble.

Reply:
@marbemac review